hack/acm-spoke.sh

```diff
@@ -0,0 +1,4 @@
+#! /usr/bin/env bash
```

Comment:
what's the file for?

Reply:
Just to simplify the spoke registration step.